examples/nat/main/config.json

```diff
@@ -8,7 +8,7 @@
             },
             "public-port": {
                 "index": 1,
-                "dst_mac": "3c:fd:fe:a4:d3:b1",
+                "dst-mac": "3c:fd:fe:a4:d3:b1",
```

Comment:
Maybe remaining dst_mac -> dst-mac too?

Reply:
dst_mac on private port should be deleted entirely since it is not used at all any more. Thanks for catching this.